### PR TITLE
Update curriculum

### PR DESCRIPTION
**Pull request by ruiasribeiro**
This PR updates the curriculum with the changes that were discussed in the last session.

### CODE REVIEW

File: apps/web/data/curriculum.ts [hunk 1]
```diff
@@ -4,92 +4,94 @@ export const BELTS: IBelt[] = [
...
+      "Compreender os conceitos e as aplicações de listas e strings",
       "Terminar 2 projetos na nova linguagem (1 deles pode ser uma adaptação de algum já feito em Scratch, se o Ninja quiser)",
-      "Apresentar um projeto a 2 mentores",
     ],
   },
   {
     colour: EBelt.Orange,
     description:
-      "Com o cinturão Laranja, o Ninja aprenderá a utilizar ferramentas da sua linguagem escolhida para criar interfaces ou até jogos simples. O Ninja deverá aprender também sobre os conceitos de I/O, com recurso às bibliotecas do sistema operativo.",
+      "Com o cinturão laranja, o ninja aprenderá a utilizar ferramentas da sua linguagem escolhida para criar interfaces ou até jogos simples. O Ninja deverá aprender também HTML, de modo a conseguir criar uma página web simples.",
```
Comment (RuiL1904) [inline apps/web/data/curriculum.ts:52]: There are places where we use "Ninja" and places where we use "ninja" (downcase). I think there should be some sort of coherence. 🙏🏼

File: apps/web/data/curriculum.ts [hunk 2]
```diff
@@ -4,92 +4,94 @@ export const BELTS: IBelt[] = [
...
+      "Para este cinturão, o Ninja aprenderá mais sobre Web e deverá decorar e adicionar interatividade ao seu próprio website.  Recomenda-se também que o Ninja possua um repositório de código para guardar os seus projetos como GitLab ou GitHub, por exemplo. Para isto, deverá aprender a trabalhar com o terminal e a usar Git. Alternativamente, poderão ser utilizadas ferramentas gráficas para introduzir o conceito de repositórios, como GitHub Desktop.",
     image: "red.png",
     requirements: [
-      "Aprender a usar uma nova linguagem que lhe permita criar o Website",
-      "Criar um Website",
-      "Apresentar um projeto para uma turma",
-      "Uma biblioteca para fazer Web servers (Flask para Python, Sinatra para Ruby, Spark para Java)",
+      "Aprender a usar CSS para alterar a estrutura e a estética de uma página HTML, com recurso a layouts flexbox e/ou grid e a animações",
+      "Aprender JavaScript para adicionar interatividade (botões, avisos, etc…) a uma página HTML",
```
Comment (RuiL1904) [inline apps/web/data/curriculum.ts:68]:
```suggestion
      "Aprender JavaScript para adicionar interatividade (botões, avisos, etc) a uma página HTML",
```

File: apps/web/data/curriculum.ts [hunk 3]
```diff
@@ -4,92 +4,94 @@ export const BELTS: IBelt[] = [
...
+      "Conhecer os utilitários básicos de linha de comandos (cd, mkdir, ls/dir, rm, cat, …)",
+      "Conhecer os comandos básicos de Git (clone, add, commit, push, pull, …), em terminal ou em interface gráfica",
```
Comment (RuiL1904) [inline apps/web/data/curriculum.ts:70]:
```suggestion
      "Conhecer os utilitários básicos de linha de comandos (cd, mkdir, ls/dir, rm, cat, etc)",
      "Conhecer os comandos básicos de Git (clone, add, commit, push, pull, etc), em terminal ou em interface gráfica",
```

File: apps/web/data/curriculum.ts (hunk 1 above)
Comment (ruiasribeiro) [reply apps/web/data/curriculum.ts]: You're right! We missed that one.

**Review by ruioliveira02 — APPROVED** [review]
Looks good to me :rocket:

I think we can do something a little bit more interesting with this page in the future, what do you think?

**Comment by ruiasribeiro** [comment]
> I think we can do something a little bit more interesting with this page in the future, what do you think?

Yeah, it does seem a bit bare-bones as it is right now. Do you have any idea in mind?

**Comment by ruioliveira02** [comment]
> > I think we can do something a little bit more interesting with this page in the future, what do you think?
> 
> Yeah, it does seem a bit bare-bones as it is right now. Do you have any idea in mind?

A tabbed view where one tab is one belt, and has everything layed out in a better form than this

**Pull request merged**